Directory.Build.targets

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think we can also delete the <?xml ?> entries, right @ericstj ?

Reply:
Good point.  MSBuild doesn't require this anymore.  I guess it depends on wether or not all our tools which try to edit these are happy without the prolog.  Seems like we'd be ok removing it and see what breaks and add it back.  The only things that might break are maestro-like tools I think, or any crusty dev tools that are accessing projects without using MSBuild.

Reply:
Good to know, I wasn't sure if we still need them. Let's see.